test/partition-healing-tests.js

```diff
@@ -395,7 +486,7 @@ function addMoreFakeNodes(t, tc, n) {
 }
 
 function validateHealRequest(t, tc, eventBody) {
-    t.equal(1, eventBody['targets'].length,
-            "/join was sent to one node as part of heal");
+    t.equal(eventBody['targets'].length, 1,
+            "expected 1 join target as part of the heal");
```

Comment:
Is this what we discussed (heal should stop upon one fail?)

The new proposal is to stop after 10 failures to offer some resilience against nodes that are truely faulty.